hadoop-tools/hadoop-sls/src/main/java/org/apache/hadoop/yarn/sls/web/SLSWebApp.java

```diff
@@ -54,6 +54,7 @@
 @Unstable
 public class SLSWebApp extends HttpServlet {
   private static final long serialVersionUID = 1905162041950251407L;
+  private static final Charset UTF8 = Charset.forName("UTF-8");
```

Comment:
If you use `Charset.forName(String)`, you need to catch some exceptions. We can avoid catching exceptions by using `StandardCharsets.UTF-8`.